setup.py

```diff
@@ -70,5 +46,25 @@ def check_python_version():
     ],
     packages=find_packages(exclude=["tests", "tests.*", "tests/*"]),
     include_package_data=True,
-    package_data={'version': [os.path.join('graspy', 'version', 'version.txt')]},  # TODO: #454 Also needs changed by https://github.com/microsoft/graspologic/issues/454,
+    package_data={
+        "version": [os.path.join("graspy", "version", "version.txt")]
+    },  # TODO: #454 Also needs changed by https://github.com/microsoft/graspologic/issues/454
+    install_requires=[
+        "networkx>=2.1",
+        "numpy>=1.8.1",
+        "scikit-learn>=0.19.1",
+        "scipy>=1.4.0",
+        "seaborn>=0.9.0",
+        "matplotlib>=3.0.0",
```

Comment:
thoughts on making [this change](https://github.com/microsoft/graspologic/blob/ef24d5ba952536578c78d2091d3f89810a61b8e0/setup.py#L27) from #421 
matplotlib 3.3.1 has a known bug with seaborn